symplyphysics/laws/gravity/vector/acceleration_due_to_gravity_via_gravity_force_and_centripetal_acceleration.py

```diff
@@ -0,0 +1,98 @@
+from sympy import Expr
+from symplyphysics import (
+    symbols,
+    units,
+    angle_type,
+    Quantity,
+    Vector,
+    QuantityVector,
+    validate_input,
+    validate_output,
+    scale_vector,
+    add_cartesian_vectors,
+    subtract_cartesian_vectors,
+    cross_cartesian_vectors,
+    vector_magnitude,
+)
+
+# Description
+## The acceleration due to gravity is the acceleration a body in the gravity field of another body (e.g. Earth)
+## with rotational effects such as the centripetal acceleration taken into account. It is the same for all bodies
+## at a fixed point, but can have different magnitude and direction at different points in space.
+
+# Law: g = F_gravity / m - cross(w, cross(w, r))
```

Comment:
Maybe it's better to use it with centripetal acceleration symbol? And let user find proper way to calculate centripetal acceleration using any of the existing formulas.

Reply:
I agree